Mu2eG4/src/constructHall.cc

```diff
@@ -113,6 +115,7 @@ namespace mu2e {
     AntiLeakRegistry& reg = art::ServiceHandle<Mu2eG4Helper>()->antiLeakRegistry();
 
     OrientationResolver* OR = new OrientationResolver();
+    OrientationResolver* ORH = new OrientationResolver();
```

Comment:
I know that you inherited this but I would like you to clean it up.  Please change this to just making an object on the stack. 

OrientationResolver ORH;

The follow on changes are:

1. ORH-> to ORH.  in one place
2. Remove the if ( 0 !=ORH) block at the end